docs/install.rst

```diff
@@ -22,6 +22,16 @@ Distribution                         Installing
                                         # pacaur -S streamlink-git
 
                                      `Installing AUR packages`_
+`Debian (sid, testing)`_             .. code-block:: console
+
+                                        # apt update
+                                        # apt install streamlink
+`Debian (stable)`_                   .. code-block:: console
+
+                                        # wget -qO - "https://bintray.com/user/downloadSubjectPublicKey?username=amurzeau" | apt-key
+                                        # deb https://dl.bintray.com/amurzeau/streamlink-debian stretch-backports main
```

Comment:
I think we should be explicit here and not just tell the user the contents of the `/etc/apt/sources.list.d/streamlink.list` file which he has to create on his own.
```bash
echo "deb https://dl.bintray.com/amurzeau/streamlink-debian stretch-backports main" | sudo tee "/etc/apt/sources.list.d/streamlink.list"
```
I'm not sure though how it'll look on the rendered page. Need to check this first...

Reply:
You are right too

Reply:
> I'm not sure though how it'll look on the rendered page.

I knew this was gonna happen. Long lines of text are adding a horizontal scroll bar to the list. I had the same issue recently when I updated the Windows installer docs. That's so stupid... Do we need to find a different solution here, or should we keep this? The other lists also have scroll bars, but those are rather short and the lists themselves are not as important as this one.
![screenshot from 2018-02-02 19-31-25](https://user-images.githubusercontent.com/467294/35748910-eb2963c8-084f-11e8-8b25-bc26663c8c01.png)